keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -1229,7 +1252,7 @@ retain the flexibility of selectors.
 
 ### Export via annotation
 
-`ServiceExport` as described has no spec and seems like it could just be
+`ServiceExport` initially had no spec and seemed like it could just be
```

Comment:
Out of scope for this change, but I think the original rationale here is actually outdated - the `Service` resource does have a `status.conditions` field now https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#ServiceStatus

Reply:
This is about ServiceExport spec so I would say it should still be right?

Reply:
This section is describing why a `ServiceExport` resource with no `spec` was originally proposed at all - the `ServiceExport` status was intended to fill a UX gap from `Service` lacking a `status.conditions` field to message if an annotation-based export would have been successful or conflicted. It's not directly applicable to the focus of this PR though, I may update the language for historical purposes in a separate PR.